code/controllers/subsystem/research.dm

```diff
@@ -24,8 +24,8 @@ SUBSYSTEM_DEF(research)
 	var/list/errored_datums = list()
 	var/list/point_types = list()				//typecache style type = TRUE list
 	//----------------------------------------------
-	var/list/single_server_income = list(TECHWEB_POINT_TYPE_GENERIC = 54.3)
-	var/multiserver_calculation = FALSE
+	var/list/single_server_income = list(TECHWEB_POINT_TYPE_GENERIC = 250)
+	var/multiserver_calculation = TRUE
```

Comment:
>build multiple servers
>get max rnd within 30 min into the game because of >>250 points per second

Reply:
also the
`//Aiming for 1.5 hours to max R&D
 	//[88nodes * 5000points/node] / [1.5hr * 90min/hr * 60s/min]
 	//Around 450000 points max???
`
is not accurate, it's just an estimate